.changeset/shaggy-knives-exercise.md

```diff
@@ -0,0 +1,9 @@
+---
+"@apollo/query-planner": patch
+"@apollo/gateway": patch
+---
+
+Adds debug/testing query planner options (`debug.bypassPlannerQueryForSingleSubgraph`) to bypass the query planning
```

Comment:
I think this should be `debug.bypassPlannerForSingleSubgraph`
```suggestion
Adds debug/testing query planner options (`debug.bypassPlannerForSingleSubgraph`) to bypass the query planning
```

Reply:
Good catch, thanks.